packages/integrations/cloudflare/src/utils/handler.ts

```diff
@@ -58,6 +64,28 @@ export async function handle(
 
 	const locals: Runtime = {
 		cfContext: context,
+		runtime: {
```

Comment:
How about using `Object.defineProperty` with `enumerable: false` to define `runtime`, so it doesn't show up on the objects